providers/dnsimple/dnsimple.md

```diff
@@ -0,0 +1,31 @@
+## Dnsimple Provider
```

Comment:
You can probably omit this file. Some other providers have them from before we made a seperate docs section.